lib/transover_core.js

```diff
@@ -36,12 +35,12 @@ function reloadAndApplyOptionsOnTabSwitch (loadAndApplyOptions) {
 Core.callAPI = (word, onresponse, sendResponse) => {
 
   const options = {
-    url: 'http://sapir.artsrn.ualberta.ca/cree-dictionary/_translate-cree/'+word,
+    url: 'https://sapir.artsrn.ualberta.ca/cree-dictionary/click-in-text?q=' + word,
```

Comment:
I'm not 100% sure, but you _might_ need to encode the `word` using [`encodeURIComponent()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/encodeURIComponent)

```suggestion
    url: 'https://sapir.artsrn.ualberta.ca/cree-dictionary/click-in-text?q=' + encodeURIComponent(word),
```

Reply:
Actually, the more I look at this, the more I'm convinced you _do_ need `encodeURIComponent()` here...